### PR TITLE
Add more descriptive personal course page

### LINKED ISSUE #10: FEATURE: More descriptive personal courses page
### Have you updated the title?

- [X] I have updated the issue's title.

### As a...

user

### I can...

view all courses

### So that...

know what courses i have

### Discussion

Please make the "All courses" display page more descriptive. Currently it only shows course ID, enroll code, & personal schedule ID. Please modify this table to display the personal schedule name, course name, and quarter, as well as any other information that may be useful.

### Acceptance Criteria

- [x] The "All courses" table shows more useful/descriptive information
- [x] I can view each course's name, schedule name, and quarter from the "All courses" table

### Implementation Todos

Frontend:
- [x] Add columns to the PS Course table for course name, schedule name, and quarter.
- [x] Format these fields correctly (i.e. quarter should be displayed in QYY format, not YYYYQ format)

Testing:
- [x] Front end tests pass and there is adequate coverage
- [x] Back end tests pass and there is adequate coverage

### PR DESCRIPTION
**Pull request by bigjr-mkkong**
This PR close issue #10 
Now the personal course page can display more information(include personal scheduler name, course name, and quarter in correct format)

Before:
image

After:
[http://proj-course-michael-dev.dokku-07.cs.ucsb.edu](http://proj-course-michael-dev.dokku-07.cs.ucsb.edu)
image

### CODE REVIEW

**Review by chloeqta — DISMISSED** [review]
LGTM

**Review by AnikaArora — DISMISSED** [review]
Please add a small description in your PR. Also, add a "before" screenshot too so we can see exactly how your code changed the original version.

**Comment by bigjr-mkkong** [comment]
> Please add a small description in your PR. Also, add a "before" screenshot too so we can see exactly how your code changed the original version.

I've updated my PR with a small description and before-after screenshots to clarify the changes I've made in this PR, please consider to merge.

**Review by bwr02 — DISMISSED** [review]
LGTM

File: src/main/java/edu/ucsb/cs156/courses/controllers/PSCourseController.java [hunk 1]
```diff
@@ -147,6 +148,23 @@ public ArrayList<PSCourse> postCourses(
       throw new IllegalArgumentException("class exists in schedule");
     }
 
+    String quarter_QYY;
```
Comment (tallyhawley) [inline src/main/java/edu/ucsb/cs156/courses/controllers/PSCourseController.java]: Why are we storing the quarter in QYY format? There is a helper on the frontend to convert from YYYYQ to QYY, just keep it in YYYYQ on the backend for consistency.

Comment (bigjr-mkkong) [reply src/main/java/edu/ucsb/cs156/courses/controllers/PSCourseController.java]: Fixed it, please consider to merge into main branch

**Review by bwr02 — APPROVED** [review]
LGTM

**Review by sophiattran — APPROVED** [review]
lgtm

**Review by tallyhawley — APPROVED** [review]
Looks good to me:)

**Pull request merged**